sentry_sdk/integrations/launchdarkly.py

```diff
@@ -0,0 +1,73 @@
+from typing import TYPE_CHECKING
+import sentry_sdk
+
+from sentry_sdk.integrations import DidNotEnable, Integration
+
+try:
+    import ldclient
+    from ldclient.hook import Hook, Metadata
+
+    if TYPE_CHECKING:
+        from ldclient import LDClient
+        from ldclient.hook import EvaluationSeriesContext
+        from ldclient.evaluation import EvaluationDetail
+
+        from sentry_sdk._types import Event, ExcInfo
+        from typing import Any, Optional
+except ImportError:
+    raise DidNotEnable("LaunchDarkly is not installed")
+
+
+class LaunchDarklyIntegration(Integration):
+    identifier = "launchdarkly"
+
+    def __init__(self, client=None):
+        # type: (LDClient | None) -> None
+        """
+        @param client    An initialized LDClient instance. If a client is not provided, this integration will attempt to
+                         use the shared global instance. This will fail if ldclient.set_config() hasn't been called.
+
+        Docs reference: https://docs.launchdarkly.com/sdk/server-side/python
```

Comment:
This needs to be linked in our docs (which can be changed) not in the code base (which is static for the life of the install).